pkg/registry/data/registry.json

```diff
@@ -466,6 +466,56 @@
       ],
       "transport": "stdio"
     },
+    "arxiv-mcp-server": {
+      "description": "Enables AI assistants to search and access arXiv research papers through MCP protocol. Mount a volume to /arxiv-papers for persistent paper storage.",
```

Comment:
The new entries don't have metadata, I don't think that's intentional?

Reply:
not intentional, let me run regup and fix this :)